src/main/java/uk/gov/pay/products/model/Payment.java

```diff
@@ -91,4 +95,12 @@ public Long getAmount() {
     public void setAmount(Long amount) {
         this.amount = amount;
     }
+
+    public String getGovUkStatus(){
+        return govUkStatus;
+    }
+
+    public void setGovukStatus(String status){
```

Comment:
minor: if you have constructor injection, then you normally not need a setter.

Reply:
👍